target/linux/ipq806x/Makefile

```diff
@@ -20,7 +21,8 @@ DEFAULT_PACKAGES += \
 	kmod-leds-gpio kmod-gpio-button-hotplug swconfig \
 	kmod-ata-core kmod-ata-ahci kmod-ata-ahci-platform \
 	kmod-usb-ohci kmod-usb2 kmod-usb-ledtrig-usbport \
-	kmod-usb3 kmod-usb-dwc3-of-simple kmod-usb-phy-qcom-dwc3 \
+	kmod-usb-dwc3-of-simple kmod-usb-phy-qcom-dwc3 \
```

Comment:
Having `kmod-usb-dwc3-of-simple` here will  probably break imagebuilder builds with 4.19, since `kmod-usb-dwc3-of-simple' isn't available there. But if `kmod-usb-phy-qcom-dwc3` depends on it for 4.14 (as it currently does), you should be able to remove `kmod-usb-dwc3-of-simple` without any issues.

Reply:
```
$ make image PROFILE=netgear_r7500
Checking 'working-make'... ok.
Checking 'case-sensitive-fs'... ok.
...
Configuring urngd.
Configuring ppp-mod-pppoe.
Collected errors:
 * opkg_install_cmd: Cannot install package kmod-usb-dwc3-of-simple.
make[2]: *** [Makefile:157: package_install] Error 255
make[1]: *** [Makefile:118: _call_image] Error 2
make: *** [Makefile:197: image] Error 2
$
```